integration-tests/src/test/java/fr/pilato/elasticsearch/crawler/fs/test/integration/AbstractITCase.java

```diff
@@ -166,6 +166,7 @@ public static void copyResourcesToTestDir() throws IOException {
 
         // We copy files from the src dir to the temp dir
         copyTestDocumentsToTargetDir(testResourceTarget, "documents", "/fscrawler-test-documents-marker.txt");
+        copyTestDocumentsToTargetDir(testResourceTarget,".", "/fscrawler-test-documents-marker.txt");
```

Comment:
Change `copyTestDocumentsToTargetDir(testResourceTarget,".", "/fscrawler-test-documents-marker.txt");` to `copyTestDocumentsToTargetDir(testResourceTarget, "tags", "/fscrawler-test-documents-marker.txt");`